apps/src/blockTooltips/DropletFunctionTooltip.html.ejs

```diff
@@ -1,13 +1,15 @@
 <div class="function-name">
   <% if (signatureOverride) { -%>
     <%= signatureOverride %>
+  <% } else if (isProperty) { -%>
+    <% if (tipPrefix) { %><%= tipPrefix %><% } %><%= functionName %>
```

Comment:
(a) is true, but (b) is not. It just puts the strings together - there is no requirement that a `.` is part of the prefix.